src/h5web/vis-packs/core/heatmap/HeatmapMesh.tsx

```diff
@@ -96,9 +99,10 @@ function HeatmapMesh(props: Props) {
     },
     vertexShader: `
       varying vec2 coords;
+      uniform float originY;
 
       void main() {
-        coords = uv;
+        coords = vec2(uv[0], abs(uv[1] - originY));
```

Comment:
The shader does not use the scale and must be inverted "manually".

As a reminder, `uv` are the normalized coordinates that vary between 0 and 1. With this formula, when `originY = 1`, it will flip the coordinates: 1 will become 0, 0 will become 1, 0.7 will become 0.3 and so on... 

When `originY = 0`, there is no change to `coords` as `uv` are always positive.

Reply:
Great!